_sub/security/external-secrets/main.tf

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------
+# Create YAML files to be picked up by Flux CD
+# --------------------------------------------------
+resource "github_repository_file" "external-secrets_helm" {
+  repository          = var.repo_name
+  branch              = local.repo_branch
+  file                = "${local.cluster_repo_path}/${local.app_install_name}-helm.yaml"
+  content             = local.app_helm_path
+  overwrite_on_create = var.overwrite_on_create
+}
+
+resource "github_repository_file" "external-secrets_helm_install" {
+  repository          = var.repo_name
+  branch              = local.repo_branch
+  file                = "${local.helm_repo_path}/kustomization.yaml"
+  content             = local.helm_install
+  overwrite_on_create = var.overwrite_on_create
+}
+
+resource "github_repository_file" "external-secrets_helm_patch" {
+  repository          = var.repo_name
+  branch              = local.repo_branch
+  file                = "${local.helm_repo_path}/patch.yaml"
+  content             = local.helm_patch
+  overwrite_on_create = var.overwrite_on_create
+}
+
+# resource "github_repository_file" "external-secrets_config_path" {
```

Comment:
While I don't disagree with the thought we need to be consistent because we set the version on most other stuff in QA, I can just see that with the recent grafana_agent we don't

While I agree we should do it with minor and patch versions (which we can with flux with the 1.x.x notation but not easily with Terraform) I don't think we should do it automatically with major so I would actually suggest we don't do it until we have our stuff in Flux.